test/test-govet.sh

```diff
@@ -65,6 +65,18 @@ function consistent-imports() {
 	if grep $'\t"github.com/purpleidea/mgmt/engine/util"' "$1"; then	# import as engineUtil
 		return 1
 	fi
+
+	# import as golang.org/x/sys/unix
+	if grep -E "^[[:space:]]+.syscall." "$1"; then
```

Comment:
Why is this DOT syscall DOT ? (what's the leading period for?)

Perhaps it's also worth it to match as done in the import example to guard against aliases too?

Reply:
Looking at this again, it doesn't make a lot of sense. I changed it to match for aliases too, check it out and see if that looks alright.

Reply:
Also suppressing output of the `grep` unless an actual bad use of `syscall` is found, then print that out with the filename.